datafusion/physical-expr/src/expressions/binary/kernels_arrow.rs

```diff
@@ -503,9 +444,6 @@ mod tests {
         let left_decimal_array = create_decimal_array(&[Some(101)], 10, 1);
         let right_decimal_array = create_decimal_array(&[Some(0)], 1, 1);
 
-        let err =
```

Comment:
Was this test removed as it is now covered by the new `arithmetic_divide_zero` test?

Reply:
No,
The behavior of divide for decimal array with decimal array was changed.

In the previous implementation, we will check the 0 for dividend, but in the kernel `divide_opt` of arrow-rs, the result will be None if the right is zero